Cargo.toml

```diff
@@ -73,6 +73,11 @@ features = [
 ]
 
 [dev-dependencies]
+dialoguer = "0.4.0"
+base64 = "0.10.1"
+directories = "2.0.2"
+mkdirp = "1.0.0"
+runtime = "0.3.0-alpha.6"
```

Comment:
I believe this can be removed. Looks good otherwise!

```suggestion
```

Reply:
Totally right, thanks!